test/test.hs

```diff
@@ -1795,14 +1796,13 @@ tests = testGroup "hevm"
           assertEqual "Must be different" (any isCex a) True
           return ()
       , testCase "eq-all-yul-optimization-tests" $ do
-        let myVeriOpts = VeriOpts{ simp = True, debug = False, maxIter = Just 5, askSmtIters = Just 20 }
-            ignoredTests =
-                    --[
-                      --"controlFlowSimplifier/terminating_for_nested.yul"
-                    --, "controlFlowSimplifier/terminating_for_nested_reversed.yul"
+        let myVeriOpts = VeriOpts{ simp = True, debug = False, maxIter = Just 5, askSmtIters = Just 20, rpcInfo = Nothing }
+            ignoredTests = [
+                    -- "controlFlowSimplifier/terminating_for_nested.yul"
+                    -- , "controlFlowSimplifier/terminating_for_nested_reversed.yul"
```

Comment:
why is this commented out code here?

Reply:
Sorry for this coment ending up in there. With the improvements we made, these actually can be checked. So some got commented out from the list of things that were ignored, but I forgot to actually delete them. Deleted now!